internal/pkg/scaffold/ansible/k8s_status.go

```diff
@@ -65,7 +65,7 @@ module: k8s_status
 
 short_description: Update the status for a Kubernetes API resource
 
-version_added: "2.7"
+version_added: "3.6"
```

Comment:
This refers to the version of Ansible, not Python

Reply:
Good catch. Tks. I will change that.